compiler/resolution/virtualDispatch.cpp

```diff
@@ -1265,6 +1272,9 @@ static void checkMethodsOverride() {
             FnSymbol* eFn = getOverrideCandidateGenericFn(fn);
             if (erroredFunctions.count(eFn) == 0) {
               if (fn->hasFlag(FLAG_OVERRIDE)) {
+               if (isDsiNewRectangularDom(fn)) {
+                // allow, for deprecation by Vass in 1.31 to implement #17131
```

Comment:
A more informative comment explaining _why_ it's allowed would be nice here: "allowed so that child classes can still override `stridable`-based versions of the procedure, which is now `strides`-based".  (?)